client/my-sites/plans-features-main/components/plan-faq.jsx

```diff
@@ -253,41 +231,21 @@ const PlanFAQ = ( { titanMonthlyRenewalCost = 0 } ) => {
 				question={ translate( 'Can I talk to a live person?' ) }
 				onToggle={ onFaqToggle }
 			>
-				{ isEnglishLocale ||
-				i18n.hasTranslation(
+				{ translate(
 					'We’d love to chat with you! All paid plans include access to one-on-one support from our ' +
 						'team of WordPress experts (we call them Happiness Engineers). The %(personalPlanName)s plan includes ' +
 						'email support while the %(premiumPlanName)s plan and above all include live chat support.{{br /}}{{br /}}' +
 						'If you have pre-purchase questions, we offer live chat on the checkout page. Select the plan ' +
 						'that looks like the best fit for you and click the “Questions? Ask a Happiness Engineer” ' +
-						'link on the next page.'
-				)
-					? translate(
-							'We’d love to chat with you! All paid plans include access to one-on-one support from our ' +
-								'team of WordPress experts (we call them Happiness Engineers). The %(personalPlanName)s plan includes ' +
-								'email support while the %(premiumPlanName)s plan and above all include live chat support.{{br /}}{{br /}}' +
-								'If you have pre-purchase questions, we offer live chat on the checkout page. Select the plan ' +
-								'that looks like the best fit for you and click the “Questions? Ask a Happiness Engineer” ' +
-								'link on the next page.',
-							{
-								args: {
-									personalPlanName: getPlan( PLAN_PERSONAL ).getTitle(),
-									premiumPlanName: getPlan( PLAN_PREMIUM ).getTitle(),
-								},
-								components: { br: <br /> },
-							}
-					  )
-					: translate(
-							'We’d love to chat with you! All paid plans include access to one-on-one support from our ' +
-								'team of WordPress experts (we call them Happiness Engineers). The Personal plan includes ' +
-								'email support while the Premium plan and above all include live chat support.{{br /}}{{br /}}' +
-								'If you have pre-purchase questions, we offer live chat on the checkout page. Select the plan ' +
-								'that looks like the best fit for you and click the “Questions? Ask a Happiness Engineer” ' +
-								'link on the next page.',
-							{
-								components: { br: <br /> },
-							}
-					  ) }
+						'link on the next page.',
+					{
+						args: {
+							personalPlanName: getPlan( PLAN_PERSONAL ).getTitle(),
+							premiumPlanName: getPlan( PLAN_PREMIUM ).getTitle(),
```

Comment:
I feel these variable names, like `personalPlanName` should have been more generic. Local variables will likely be a lot harder to refactor down the line than constant slugs like `PLAN_PERSONAL`. I'm also not sure if it will actually trigger a re-translation if we change replacement variables in translate calls.

This is also the case for single-variable replacements I've seen. In those cases, would a simple `planName` not suffice? For here, it could have been anything more generic (even like an array - or planONE, planTWO, etc.). It feels "backwards" when we want to move away from "business" and "personal" and use those in variables (of any type or form).

Reply:
> In those cases, would a simple planName not suffice?

This could work, but we'll need to mention the plan name in the comment - some translations may depend on the gender of the plan name.

Keeping the old plans in the name made more sense when the changes were meant as an experiment, where other plans could be tested before committing to one.

Reply:
Good callout. Keeping the variable names more generic + a translator comment should be a better way to go, especially now that it's no longer an experiment. I think updating it now or not depends on whether it'd become a new string that needs to be translated.

@dlind1 could you help me confirm if that's the case? I'd be grateful 🙇🏼

Reply:
Actually, we can first deploy this and do a follow-up if we decide to update the placeholder variable name :)

Reply:
Sure. Follow-up fine. Most other cases I refactored use `planName`. It's only these couple that don't. I wouldn't mention "personal" or "business" in the translator comments either (as I've seen done on several occasions in these few PRs). Those names are effectively redundant/irrelevant right now. Use "Explorer" or "Foo" if you need to.

e.g. does mention of "Personal" help a translator when "Personal" is no longer a thing?

> /* translators: %(planName)s is the short-hand version of the Personal plan name */

Reply:
@southp
> I think updating it now or not depends on whether it'd become a new string that needs to be translated.

Placeholders are treated as part of the string, so changing it would either result in a completely new original, or a fuzzy match if the string is long enough and the difference between the old and new strings is small enough. A new original would be translated from scratch, while a fuzzy match would reuse the existing translations.

Providing more details to the translators (like which plan names could be in the placeholder) is always useful to improve the quality and accuracy of the translations, although it'd be even better to avoid using translated text in placeholders, if possible.